docs/using_unidist/unidist_on_mpi.rst

```diff
@@ -142,42 +158,14 @@ Second, by setting the configuration value associated with the environment varia
 
 Running is the same as in a single node.
 
-SPMD model
-""""""""""
-
-First of all, to run unidist on MPI in a cluster using `SPMD model <https://en.wikipedia.org/wiki/Single_program,_multiple_data>`_,
-you should set the ``UNIDIST_IS_MPI_SPAWN_WORKERS`` environment variable to ``False``:
+.. note::
+    Root proccess will allways be executed locally and other proccesses will be spawned in order on the specified hosts.
+    If you want to run root proccess on anoother host you should use `ssh host` before your command and thoroughly check that environment will be correct. 
+    You can set some variables into ssh or activate conda envirenment before running python script.
+    
+    If you want to start a root process on another host, you should use `ssh host` before the command and carefully check that the environment is correct. 
+    You can set some variables in ssh command or activate the conda environment right before running the Python script:
 
 .. code-block:: bash
 
-    $ export UNIDIST_IS_MPI_SPAWN_WORKERS=False
```

Comment:
Why is everything below removed?

Reply:
because the structure has changed and there is no need to write about the SPMD model here